BlakeDonn/ideabox-boilerplate
# Add new card functionality

### What’s this PR do? 

* Linked and created our main.js and idea.js. 

* Created a class constructor of ideas. 

* Created function that allows user to create their own cards

*  Created active and inactive functionality for save idea button 

### Where should the reviewer start? 

* Start at the idea.js and ensure it is structured correctly

* Check the entirety of main.js

* Check the HTML element attributes

### How should this be manually tested?

Enter information into both input fields, press save button, repeat. Check that the styling of the button changes when adding or removing text from the input fields. Check for edge-cases to break the code.

### What are the relevant tickets?

We are driver-navigating this request, we have an outstanding CSS styling refactor that is currently open.

### Screenshots (if appropriate): 

![inputfields](https://i.imgur.com/KQ79DVG.png)
![oneidea](https://i.imgur.com/5aHsVeR.png)
![multipleideas](https://i.imgur.com/ka9xU6f.png)


### Questions:
